static/styles/explorer.scss

```diff
@@ -1632,3 +1632,7 @@ span.lib-info {
 .library-info-popover {
     max-width: 600px;
 }
+
+button.new-pane-button:disabled:after {
+    content: ' [already open]';
```

Comment:
Maybe a better message would be `(already opened)`? Not that big of a deal though